client/packages/system/src/Patient/api/api.ts

```diff
@@ -40,7 +40,8 @@ export const getPatientQueries = (sdk: Sdk, storeId: string) => ({
         desc: !!sortBy?.isDesc,
         storeId,
         filter: {
-          isCustomer: true,
+          isStore: false,
```

Comment:
Just picking something that means I get some data 😉 
Another one that needs updating